tests/cleanup/remove_old_objects_openstack.sh

```diff
@@ -65,9 +65,9 @@ rlJournalStart
       name: "{{item.id}}"
       state: absent
     loop: "{{openstack_servers}}"
-    when: item.created < lookup('env','TIMESTAMP')
+    when: item.created < lookup('env','TIMESTAMP') and item.metadata.Tag != "keep_me"
```

Comment:
ansible fails with an error if item.metadata.Tag is not defined. It can be fixed this way:
when: item.created < lookup('env','TIMESTAMP') and (item.metadata.Tag is not defined or item.metadata.Tag != "keep_me")